ui/src/app/machines/components/ActionFormWrapper/DeployForm/DeployForm.test.tsx

```diff
@@ -386,4 +382,96 @@ describe("DeployForm", () => {
     ]);
     mockUseSendAnalytics.mockRestore();
   });
+
+  it("can register a LXD VM host", () => {
+    const state = { ...initialState };
+    state.machine.selected = ["abc123"];
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter
+          initialEntries={[{ pathname: "/machines", key: "testKey" }]}
+        >
+          <DeployForm setSelectedAction={jest.fn()} />
+        </MemoryRouter>
+      </Provider>
+    );
+    act(() =>
+      wrapper.find("Formik").props().onSubmit({
+        kernel: "",
+        oSystem: "ubuntu",
+        release: "bionic",
+        vmHostType: PodType.LXD,
+      })
+    );
+    expect(
+      store.getActions().filter((action) => action.type === "machine/deploy")
+    ).toStrictEqual([
```

Comment:
Do you think this should just check that the params include `register_vmhost` but not `install_kvm` rather than the whole object?

Reply:
Good idea!